### PR TITLE
feat: remove thebutton from blogs and fix broken links

### PR DESCRIPTION
**Pull request by yrobla**
Closes: #1926

### CODE REVIEW

File: site/blog/unlisted/deploy-ghost-ec2.mdx [hunk 1]
```diff
@@ -146,7 +152,7 @@ SELECT
 SELECT
   *
 FROM
-  iasql_commit();
+  iasql_commit ();
```
Comment (depombo) [inline site/blog/unlisted/deploy-ghost-ec2.mdx:155]: does that still work?

**Review by depombo — APPROVED** [review]

File: site/blog/unlisted/deploy-ghost-ec2.mdx (hunk 1 above)
Comment (dfellis) [reply site/blog/unlisted/deploy-ghost-ec2.mdx:155]: Yes, though I don't know why you would format that way.

Comment (yrobla) [reply site/blog/unlisted/deploy-ghost-ec2.mdx:155]: it's just prettier formatting it

**Review by mtp1376 — APPROVED** [review]

**Review by aguillenv — APPROVED** [review]
not a huge fan of the prettier SQL formatting but we can tackle that later

**Pull request merged**